app/views/mailers/digest_mailer/daily_dev_email.html.erb

```diff
@@ -0,0 +1,71 @@
+
+
+<h1 style="text-align: center;">
+  DEV Daily Digest
+</h1>
```

Comment:
Can we add some tech-focused emojis in here to inject a little more fun? Thinking a row under the header?